src/wallet/asyncrpcoperation_mergetoaddress.h

```diff
@@ -25,10 +26,12 @@
 using namespace libzcash;
 
 // Input UTXO is a tuple of txid, vout, amount
```

Comment:
update comment (for Cscript)?

Reply:
Whoops... missed that. I was tempted to just delete some of these comments because I think that these tuples are self-explanatory. That being said, I'll update it to add `, script`.